src/components/ClockModule.js

```diff
@@ -0,0 +1,281 @@
+import React, { useCallback, useRef, useState } from 'react'
+import {
+  blockExplorerUrl,
+  Button,
+  ButtonBase,
+  GU,
+  IconCheck,
+  IconClock,
+  IconCross,
+  IconExternal,
+  textStyle,
+  Timer,
+  useTheme,
+} from '@aragon/ui'
+import { useWallet } from 'use-wallet'
+
+import HeaderModule from './Header/HeaderModule'
+import HeaderPopover from './Header/HeaderPopover'
+import useNetwork from '../hooks/useNetwork'
+import { useHeartbeat } from '../hooks/useCourtContracts'
+import { useCourtClock } from '../providers/CourtClock'
+import { useCourtConfig } from '../providers/CourtConfig'
+
+import { shortenAddress } from '../lib/web3-utils'
+import { formatDuration } from '../utils/date-utils'
+
+import logoSvg from '../assets/LogoAccent.svg'
+
+function ClockModule() {
+  const buttonRef = useRef()
+  const [opened, setOpened] = useState(false)
+
+  const theme = useTheme()
+  const wallet = useWallet()
+  const onHeartbeat = useHeartbeat()
+  const courtConfig = useCourtConfig()
+  const { type: networkType } = useNetwork()
+  const {
+    currentTermId,
+    currentTermEndDate,
+    isSynced,
+    neededTransitions,
+  } = useCourtClock()
+
+  const toggle = useCallback(() => setOpened(opened => !opened), [])
+
+  const handlePopoverClose = useCallback(() => {
+    setOpened(false)
+  }, [])
+
+  const handleOnClick = useCallback(async () => {
+    try {
+      const tx = await onHeartbeat(neededTransitions)
+      await tx.wait()
+    } catch (err) {
+      console.error(err)
+    }
+  }, [neededTransitions, onHeartbeat])
+
+  const IconSync = isSynced ? IconCheck : IconCross
+
+  return (
+    <div
+      ref={buttonRef}
+      tabIndex="0"
+      css={`
+        display: flex;
+        align-items: center;
+        height: 100%;
+        outline: 0;
+      `}
+    >
+      <HeaderModule
+        icon={
+          <div
+            css={`
+              background-color: ${theme.background};
+              display: flex;
+              border-radius: 100px;
+              padding: 2px;
+              margin-right: ${0.5 * GU}px;
+            `}
+          >
+            <IconClock
+              color={isSynced ? theme.positive : theme.negative}
+              width={4.5 * GU}
+              height={4.5 * GU}
+            />
+          </div>
+        }
+        content={
+          <div
+            css={`
+              line-height: 1;
+              width: 120px;
+
+              & > time {
+                & > span:first-child {
+                  display: none;
+                }
+                & * {
+                  line-height: 1;
+                }
```

Comment:
Same here, ideally we should either add a mode on the component or try to find another way to achieve this.